custom/enikshay/integrations/nikshay/field_mappings.py

```diff
@@ -116,9 +116,9 @@
     'SC-7': 7,
     'SC-8': 8,
     'SC-9': 9,
-    '1+': 11,
```

Comment:
Instead of changing the existing mapping, can we add new mappings? That way older data will also continue to work.

Reply:
@sheelio thank you for noting that.. adding back old mapping as well